encrypt-actions/index.js

```diff
@@ -77,17 +78,24 @@ async function decrypt(action, key) {
   return bytesToObj(bytes)
 }
 
-export function encryptActions(client, secret, opts = {}) {
+export async function encryptActions(client, secret, opts = {}) {
```

Comment:
Can we avoid making it `async`?

I'm afraid that some user may do a mistake like writing this code:

```js
encryptActions(client, secret)
client.start() // => connection will be started without encryption
```

Reply:
yes, that's a good idea.

This small change of mine has turned into a difficult merge. Is it ok if I close this PR, and I can just depend on my forked library?

Reply:
Let’s fix this https://github.com/logux/client/pull/105#discussion_r1495001558

and then I will merge it and fix `async` by my own. The change is useful and we need to keep your name in the project history :).

Reply:
👍